subsys/event_manager/event_manager.c

```diff
@@ -309,6 +305,18 @@ void _event_submit(struct event_header *eh)
 
 int event_manager_init(void)
 {
+	__ASSERT_NO_MSG(__stop_event_types - __start_event_types <=
+			CONFIG_EVENT_MANAGER_MAX_EVENT_CNT);
+
+	#if CONFIG_EVENT_MANAGER_PROFILER_ENABLED
```

Comment:
Preprocessor conditionals should be aligned to line start.

Why can't we use `IS_ENABLED` here? because of CONFIG_PROFILER_MAX_NUMBER_OF_EVENTS? 

In such cases I prefer to slightly complicate Kconfig file to get rid of preprocessor.
When profiler is not enabled, you can simply create a noprompt config PROFILER_MAX_NUMBER_OF_EVENTS that would define to zero. Please do that or fix the alignment.

Reply:
Ok, I've fixed alignment.